main.cpp

```diff
@@ -10,6 +10,21 @@
 
 #include <LyricListManager.h>
 
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+#ifdef Q_OS_WIN
+// On Windows, SDL will do initialization in "SDL_main()" before calling our "main()".
```

Comment:
这句注释是否有改进的空间呢？因为 [main 本身被定义为 SDL_main](https://hg.libsdl.org/SDL/file/d4c15ecfcf62/include/SDL_main.h#l109)

或许可以参考 [SDL_main.h 文件里的一句话](https://hg.libsdl.org/SDL/file/d4c15ecfcf62/include/SDL_main.h#l35)，
> On Windows SDL provides WinMain(), which parses the command line and passes the arguments to your main function

改为类似：
```suggestion
// On Windows SDL provides WinMain(), which parses the command line and passes the arguments to our main function
```